Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && \
 
 RUN npm prune --production
 
-FROM node:14-buster-slim
+FROM node:14.17-buster-slim
```

Comment:
`14` will pick up `14.17` anyway, don't think this is required

Reply:
Well it's best to be consistent, but also the security alert isn't going away by just re-deploying so i thought this might sort it.

I was in the process of checking the docker image itself, but I have had to address some other stuff (e.g App Insights), which is why I hadn't asked for this to be reviewed yet.

Reply:
Well, I can't see why this vuln is reported here but not in e.g. `hmpps-restricted-patients-prod` docker image.

Doing a `dpkg -l | grep zlib` in all of these images gives:

zlib1g:amd64            1:1.2.11.dfsg-1        amd64        compression library - runtime

So hopefully by doing this update it will stop alerting 🤷

Reply:
I can see from https://hub.docker.com/_/node that
```
14-buster-slim, 14.19-buster-slim, 14.19.1-buster-slim, fermium-buster-slim
```
are all the same docker image.  If you're wanting to pin to anything better to pin to the latest 14.19 instead.

Reply:
Ah yes - ok will do